src/Moryx.AbstractionLayer/Resources/IResourceGraph.cs

```diff
@@ -76,11 +76,5 @@ TResource Instantiate<TResource>(string type)
         /// Remove resource, but only flag it deleted
         /// </summary>
         bool Destroy(IResource resource);
-
-        /// <summary>
-        /// Remove a resource permanently and irreversible
-        /// </summary>
-        [Obsolete("Permanent removal of resources will be removed in the next major")]
```

Comment:
@dbeuchler `Destroy(IResource resource, bool permanent)` is still implemented here

[ResourceGraph](https://github.com/PHOENIXCONTACT/MORYX-Framework/blob/337150e9f7b4aa567b9db756d2fc8a0da68619cb/src/Moryx.Resources.Management/Resources/ResourceGraph.cs#L152-L155)

and the `DestroyDelegate` set here

[ResourceManager](https://github.com/PHOENIXCONTACT/MORYX-Framework/blob/337150e9f7b4aa567b9db756d2fc8a0da68619cb/src/Moryx.Resources.Management/Resources/ResourceManager.cs#L110)

Reply:
I dont like this Obsolete! We have resources which creates resources on the fly and removes them permanently (e.g. identities), keep these resources in the database would mess it up. Who marked that as obsolete? I would vote for keeping this feature. @1nf0rmagician do you have something against that?